.travis.yml

```diff
@@ -3,20 +3,20 @@ dist: xenial
 language: scala
 env:
   global:
-  - RELEASE_SCALA_VERSION=2.12.2
+  - RELEASE_SCALA_VERSION=2.12.10
```

Comment:
What was the reason for this increment of the version?

Reply:
This was done to avoid Scala-JDK compatibility problems when using JDK11.
I checked on [docs.scala-lang.org](https://docs.scala-lang.org/overviews/jdk-compatibility/overview.html) and 2.12.4 seems the minimum 2.12 version required for JDK11, but it also says that 
> As of Scala 2.13.0, 2.12.8 and 2.11.12, JDK 11 support is incomplete

I finally saw that on [the “Support JDK 11” issue](https://github.com/scala/scala-dev/issues/139) they recently added
> In general, Scala now works just fine on JDK 11

so i decided to upgrade to scala 2.12.10, to be safe.
Should i use a different version?

Reply:
No, I'd say it is fine! Thanks